app/models/repository.rb

```diff
@@ -25,6 +25,8 @@ def self.license_options
     presence:   { message: "Project title is required."},
     uniqueness: { message: "Project title is already in use.", scope: :user_username}
 
+  # validates :photos, :length => { :minimum => 1 }
```

Comment:
You could probably add this for new repos so that it's `validates :photos, :length => { :minimum => 1 }, on: :create` (or something like that)